src/screens/Customers/EditStartingMeter.tsx

```diff
@@ -25,39 +26,41 @@ interface EditStartingMeterProps extends RouteComponentProps {
     classes: { amountOwedContainer: string; };
 }
 
+const validationSchema = yup.object({
+    startingMeterAmount: yup.number()
```

Comment:
Might also be worth checking to see if this is identical to the current starting meter and not allowing that -- currently I can make "updates" of the existing values

Reply:
Nice catch!